### PR TITLE
Member bio for Ashley Griffin

### PR DESCRIPTION
**Pull request by griffinac**
# Member bio for Ashley Griffin

## :recycle: Current situation & Problem
Ashley was missing from the class app!

## :bulb: Proposed solution
Added Ashley's bio

## :gear: Release Notes 
Ashley's bio

### Testing
Added UI test

### Reviewer Nudging
@vishnuravi or @PSchmiedmayer could you check my code? :)

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by PSchmiedmayer — CHANGES_REQUESTED** [review]
Nice job @griffinac! I like the additions!
The code looks good, the next step would be to resolved the SwiftLint warnings that you currently have in your project and that also make the SwiftLint check fail in this PR: https://github.com/CS342/CS342-2023-Application/actions/runs/3976586316/jobs/6817169167

You should see the warnings also appear in Xcode in the specific lines that cause violations. You can also see the as comments in the PR diff: https://github.com/CS342/CS342-2023-Application/pull/8/files

File: CS342MemberViews/Sources/AshleyGriffin/Resources/AshleyGriffinBio.md [hunk 1]
```diff
@@ -1 +1,7 @@
 Dr. Ashley Griffin is a medical informatics postdoctoral fellow in the Department of Health Policy at Stanford and the VA Palo Alto Health Care System. Her research focuses on the use of digital health technologies and patient-generated health data to facilitate shared decision-making and inform interoperability initiatives.
+
+**Hobbies**
+- Cross-stitch 🪡
+- Piano 🎹
+- Yoga 🧘‍♀️
+- Listening to Taylor Swift 🕛
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/AshleyGriffin/Resources/AshleyGriffinBio.md:7]: @griffinac you get extra points for this one 😊

**Review by vishnuravi — APPROVED** [review]
Nicely done @griffinac! A great example for students to follow!

**Review by PSchmiedmayer — APPROVED** [review]
Looks great @griffinac 🚀

**Comment by PSchmiedmayer** [comment]
@griffinac Feel free to resolve the conversations and then you can merge the PR 🚀

File: CS342MemberViews/Sources/AshleyGriffin/Resources/AshleyGriffinBio.md (hunk 1 above)
Comment (griffinac) [reply CS342MemberViews/Sources/AshleyGriffin/Resources/AshleyGriffinBio.md:7]: Agree that mentions of Taylor Swift deserve extra points 🤣

**Review by griffinac — COMMENTED** [review]
Updated member bio

**Pull request merged**